whelktool/scripts/syslib/Ylm-add-availability.groovy

```diff
@@ -0,0 +1,76 @@
+String where = """id in (
+   SELECT lh.id FROM lddb lh WHERE
+   lh.data #>>'{@graph,1,heldBy,@id}' = 'https://libris.kb.se/library/Ylm'
+   AND lh.deleted = 'false')
+"""
+
+selectBySqlWhere(where, silent: false, { hold ->
+
+    def holds = hold.graph[1]
+    boolean shouldChange = false
+    boolean modified = false
+
+    if (holds["hasComponent"]) {
+
+        holds["hasComponent"].each { it ->
+            if (it["availability"]) {
+                asList(it["availability"]).each { availability ->
+
+                    if (availability["label"] && availability["label"] instanceof String) {
+                        if (!availability["label"].contains("Ej fjärrlån")) {
+                            shouldChange = true
+                        }
+                    }
+                }
```

Comment:
This should be reversed?
Now `shouldChange` is true if there is _any_ availability that is _not_ "Ej fjärrlån"
It should be false if there is _any_ availability that _is_ "Ej fjärrrlån"